docs/api-references/docs.md

```diff
@@ -1401,7 +1401,8 @@ Kubernetes core/v1.Affinity
 </td>
 <td>
 <em>(Optional)</em>
-<p>Affinity of TiDB cluster Pods</p>
+<p>Affinity of TiDB cluster Pods
+Will be overwritten by each cluster component&rsquo;s specific affinity setting</p>
```

Comment:
This looks confusing...
![image](https://user-images.githubusercontent.com/59654584/101717201-c9a0b980-3ad9-11eb-95e6-fba974fbc710.png)
```suggestion
<p>Affinity of TiDB cluster Pods<br>
Will be overwritten by each cluster component&rsquo;s specific affinity setting</p>
```

Reply:
This file is auto-generated from `pkg/apis/pingcap/v1alpha1/types.go`, which is a little tricky...
Maybe we can add additional blank line comments in `types.go`, WDYT @DanielZhangQD